src/dvr/dvr_db.c

```diff
@@ -17,6 +17,8 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#define _ISOC9X_SOURCE
```

Comment:
Because it did not resolve the issue. When `configure` runs in the cross-compilation environment, it checks `llabs` as OK. Therefore, the `static inline` definition is not picked up. The cross-compilation still fails with the same old `implicit declaration` error above unless I define the `_ISOC9X_SOURCE`.

But please don't worry about this. This affects old platforms with old toolchains only. I can patch this downstream for the affected platforms only. No prob. Everything else is fixed now!